sage-ios/SAGE/SAGE/ProfileView.swift

```diff
@@ -122,8 +164,30 @@ class ProfileView: UIView {
         self.profileUserImgBorder.layer.cornerRadius = (self.profileImageSize + self.profileImageBorder)/2
         self.profileUserImgBorder.backgroundColor = UIColor.whiteColor()
         
-        self.profileEditButton.font = UIFont.normalFont
-        self.profileEditButton.textColor = UIColor.secondaryTextColor
+        self.profileEditButton.setTitle("Edit Profile", forState: .Normal)
+        self.profileEditButton.setTitleColor(UIColor.secondaryTextColor, forState: .Normal)
+        self.profileEditButton.titleLabel?.font = UIFont.normalFont
+        self.profileEditButton.titleLabel?.textColor = UIColor.secondaryTextColor
+        
+        self.promoteButton.setTitle("Promote", forState: .Normal)
+        self.promoteButton.titleLabel?.font = UIFont.normalFont
+        self.promoteButton.setTitleColor(UIColor.secondaryTextColor, forState: .Normal)
+        self.promoteButton.layer.borderWidth = 1.0
+        self.promoteButton.layer.borderColor = UIColor.secondaryTextColor.CGColor
+        self.promoteButton.layer.cornerRadius = 3
+        self.promoteButton.clipsToBounds = true
+        self.promoteButton.contentEdgeInsets = UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5)
+        self.promoteButton.hidden = false
+
+        self.demoteButton.setTitle("Demote", forState: .Normal)
+        self.demoteButton.titleLabel?.font = UIFont.normalFont
+        self.demoteButton.setTitleColor(UIColor.lightRedColor, forState: .Normal)
+        self.demoteButton.layer.borderWidth = 1.0
+        self.demoteButton.layer.borderColor = UIColor.lightRedColor.CGColor
+        self.demoteButton.layer.cornerRadius = 3
+        self.demoteButton.clipsToBounds = true
+        self.demoteButton.contentEdgeInsets = UIEdgeInsets(top: 5, left: 5, bottom: 5, right: 5)
+        self.demoteButton.hidden = true
```

Comment:
you can just use the setThemeColor(_:) method for both the promote and demote button so that the loading icon will match the color of the text